.swcrc

```diff
@@ -1,7 +1,11 @@
 {
   "$schema": "https://json.schemastore.org/swcrc",
   "jsc": {
-    "target": "es2020"
+    "target": "es2020",
+    "parser": {
+      "syntax": "typescript",
```

Comment:
swc doesn't have capability for type-checking

https://swc.rs/docs/migrating-from-tsc#notes

It just compiles ts files to js ones